openeogeotrellis/geopysparkcubemetadata.py

```diff
@@ -118,6 +118,32 @@ def get_nodata_value(self, requested_bands, default_value) -> float:
             no_data_value = default_value
         return float(no_data_value)
 
+    def get_layer_crs(self):
+        crs = self.get("cube:dimensions", "x", "reference_system", default="EPSG:4326")
+        if isinstance(crs, int):
+            crs = "EPSG:%s" % str(crs)
+        elif isinstance(crs, dict):
+            if crs["name"] == "AUTO 42001 (Universal Transverse Mercator)":
```

Comment:
This could also use a dict.get to avoid throwing an exception.

Reply:
True! Changed here: https://github.com/Open-EO/openeo-geopyspark-driver/commit/cdd731ce6d684eba894beff7c8ac78266ddf12b0